it/controllers/RegimeAPIControllerISpec.scala

```diff
@@ -180,108 +267,189 @@ class RegimeAPIControllerISpec
-//               |  "hasAnyPenaltyData": true
-//               |}
-//               |""".stripMargin
-//           )
-//         }
-//       }
-//
-//       s"return ISE (${Status.INTERNAL_SERVER_ERROR})" when {
-//         "the get penalty details call fails" in {
-//
-//           mockStubResponseForAuthorisedUser
-//           mockStubResponseForGetPenaltyDetails(Status.INTERNAL_SERVER_ERROR, regime, idType, id, body = Some(""))
-//
-//           val result = await(buildClientForRequestToApp(uri = s"/${regime.value}/summary/${idType.value}/${id.value}").get())
-//           result.status shouldBe INTERNAL_SERVER_ERROR
-//         }
-//       }
-//
-//       s"return NOT_FOUND (${Status.NOT_FOUND})" when {
-//         "the get penalty details call returns 404" in {
-//
-//           mockStubResponseForAuthorisedUser
-//           mockStubResponseForGetPenaltyDetails(Status.NOT_FOUND, regime, idType, id, body = Some(""))
-//
-//           val result = await(buildClientForRequestToApp(baseUrl = "", uri = s"/${regime.value}/summary/${idType.value}/${id.value}").get())
-//           result.status shouldBe NOT_FOUND
-//         }
-//       }
-//
-//       s"return NO_CONTENT (${Status.NO_CONTENT})" when {
-//         "the get penalty details call returns 404 (with NO_DATA_FOUND in body)" in {
-//           val notFoundResponseBody: String =
-//             """
-//               |{
-//               |  "failures": [
-//               |    {
-//               |      "code": "NO_DATA_FOUND",
-//               |      "reason": "The remote endpoint has indicated that no penalty data found for provided ID number."
-//               |    }
-//               |  ]
-//               |}
-//               |""".stripMargin
-//
-//           mockStubResponseForAuthorisedUser
-//           mockStubResponseForGetPenaltyDetails(Status.NOT_FOUND, regime, idType, id, body = Some(notFoundResponseBody))
-//
-//           val result = await(buildClientForRequestToApp(uri = s"/${regime.value}/summary/${idType.value}/${id.value}").get())
-//           result.status shouldBe NO_CONTENT
-//         }
-//
-//         "the get penalty details call returns 200 with an empty body" in {
-//           val emptyResponse: String = "{}"
-//
-//           mockStubResponseForAuthorisedUser
-//           mockStubResponseForGetPenaltyDetails(Status.OK, regime, idType, id, body = Some(emptyResponse))
-//
-//           val result = await(buildClientForRequestToApp(uri = s"/${regime.value}/summary/${idType.value}/${id.value}").get())
-//           result.status shouldBe NO_CONTENT
-//         }
-//       }
-//     }
+      Table(
+      ("Regime", "IdType", "Id"),
+      (Regime("VATC"), IdType("VRN"), Id("123456789")),
+      (Regime("ITSA"), IdType("NINO"), Id("AB123456C"))
+    ).forEvery { (regime, idType, id) =>
+
+      val enrolmentKey = AgnosticEnrolmentKey(regime, idType, id)
+
+    s"getSummaryData for $regime" when {
+      val penaltyUpstreamServices = Seq("HIP", "ETMP")
```

Comment:
All 1811 and 1812 calls still go to ETMP, 1808 still goes to PEGA. 
The change is 
before: BE -> IF platform -> ETMP API
after:   BE -> HIP platform -> ETMP API

Please change the ETMP references and mocks to IF